packages/b/buttercms.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "ButterCMS",
+    "description": "ButterCMS JS API Client",
+    "filename": "buttercms.min.js",
+    "homepage": "https://github.com/buttercms/buttercms-js",
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/buttercms/buttercms-js.git"
+    },
+    "keywords": [
+      "buttercms",
+      "butter",
+      "cms",
+      "api",
+      "headless cms",
+      "SDK",
+      "javascript"
+    ],
+    "license": "MIT",
+    "autoupdate": {
+      "source": "npm",
+      "target": "buttercms",
+      "fileMap": [
+        {
+          "basePath": "lib",
+          "files": [
+            "*.js",
+            "utils/*.js",
+            "resources/*.js"
+          ]
+        }
+      ]
+    },
+    "author": "ButterCMS <support@buttercms.com>"
```

Comment:
As the CI notes, we don't accept an `author` property, but we do accept an `authors` property that is an array of objects with a `name`/`email` property.

Reply:
> As the CI notes, we don't accept an `author` property, but we do accept an `authors` property that is an array of objects with a `name`/`email` property.

@MattIPv4 Sorry about that. I have fixed and squashed. Thanks!